resources/functionMap.php

```diff
@@ -3933,7 +3933,7 @@
 'header' => ['void', 'header'=>'string', 'replace='=>'bool', 'http_response_code='=>'int'],
 'header_register_callback' => ['bool', 'callback'=>'callable'],
 'header_remove' => ['void', 'name='=>'string'],
-'headers_list' => ['array'],
+'headers_list' => ['list<string>'],
```

Comment:
Would `list<non-empty-string>` make sense? Can a header be an empty string?

Reply:
or `non-falsy-string` even?

Reply:
```
header('');
header('0');
header('Test: test');
var_dump(headers_list());
```

ran this code and got this:

```
array (size=2)
  0 => string '0' (length=1)
  1 => string 'Test: test' (length=10)
```

so `non-empty-string` is correct